hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneVolume.java

```diff
@@ -444,6 +471,14 @@ public void deleteBucket(String bucketName) throws IOException {
     proxy.deleteBucket(name, bucketName);
   }
 
+  public long getRefCount() {
```

Comment:
Initially I added this because findbugs was complaining that refCount is unused.

I believe we could expose also refCount in `ozone volume info` / `ozone volume update`

Reply:
The getter is probably ok, but we should be able to get rid of the setter without findbugs complaining because the new constructor initializes it. Probably also a good idea to have the old constructor also set it to 0 so there is no confusion about what its default value should be.